stations.csv

```diff
@@ -26820,7 +26820,7 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;is_city;country;
 28958;Malpensa Aeroporto T2;malpensa-aeroporto-t2;8301146;;45.646821;8.723376;28988;f;IT;f;Europe/Rome;t;;;f;;f;;f;;f;;f;;f;8301146;t;;;;f;;f;;f;;f;;f;f;;Mailand Flughäfen;Milan Airport;Milán Aeropuerto;Aéroport de Milan;Aeroporto di Milano;;;;;;;;;;;;
 28959;Courmayeur;courmayeur;8313836;;45.79659;6.96893;;f;IT;f;Europe/Rome;t;;;f;;f;;f;u0huqx;t;;f;;f;8313836;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;クールマイユール;코우르마이에우르;;;Курмайор;;;库尔马耶乌尔
 28960;Madonna di Campiglio;madonna-di-campiglio;8313837;;46.22671;10.82582;;f;IT;f;Europe/Rome;t;;;f;;f;;f;;f;11488;t;;f;8313837;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;マドンナ・ディ・カンピーリオ;;;;Мадонна ди Кампильо;;;
-28961;Cortina d’Ampezzo;cortina-dampezzo;8313835;;46.5369;12.13903;;f;IT;f;Europe/Rome;t;;;f;;f;;f;;f;;f;;f;8313835;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;コルティーナ・ダンペッツォ;;;;Кортина д'Ампеццо;;;
+28961;Cortina d’Ampezzo;cortina-dampezzo;8313835;;46.5369;12.13903;;f;IT;f;Europe/Rome;f;;;f;;f;;f;;f;;f;;f;8313835;t;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;コルティーナ・ダンペッツォ;;;;Кортина д'Ампеццо;;;
```

Comment:
Also, if we disable this one, we should disable Courmayeur and Madonna di Campiglio as well.